ulog/builder.go

```diff
@@ -37,10 +38,18 @@ type Configuration struct {
 	Stdout  bool
 	Verbose bool
 
+	Sentry *SentryConfiguration
+
 	prefixWithFileLine *bool `yaml:"prefix_with_fileline"`
 	TextFormatter      *bool // use TextFormatter (default json)
 }
 
+// SentryConfiguration provides sentry DSN
+// TODO(glib): allow other sentry config stuff through yaml: trace skips, etc
+type SentryConfiguration struct {
+	DSN string
```

Comment:
It's validated upon client creation. Not sure we should just do regex here, should we?